### PR TITLE
Add cargo clippy in linting script

### PR DESCRIPTION
**Pull request by j33n**
Finishes #1150

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -85,6 +85,7 @@ install:
   rm -rf "$RUSTUP_HOME"toolchains/*/etc
   rm -rf "$RUSTUP_HOME"toolchains/*/share
 before_script:
+- rustup component add clippy-preview
```
Comment (hayd) [inline .travis.yml]: I think this needs to happen in the install step, before "   # Remove unnnecessary cargo and rustup directories."

Comment (hayd) [reply .travis.yml]: e.g. https://github.com/hayd/deno/commit/d5d40e846d2796b2f1b3251ac2454784df7886ce

Comment (j33n) [reply .travis.yml]: I'm closing this up since I'm not making any reasonable progress 😞

**Comment by hayd** [comment]
This would be good to have!

**Pull request closed without merging**